v1/src/datadog.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed
+ * under the Apache License Version 2.0.
+ *
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2021 Datadog, Inc.
+ */
+
+import * as lambda from "@aws-cdk/aws-lambda";
+import * as lambdaNodejs from "@aws-cdk/aws-lambda-nodejs";
+import * as lambdaPython from "@aws-cdk/aws-lambda-python";
+import * as logs from "@aws-cdk/aws-logs";
+import * as cdk from "@aws-cdk/core";
+import log from "loglevel";
+import { Transport } from "./common/transport";
+import {
+  applyLayers,
+  redirectHandlers,
+  addForwarder,
+  addForwarderToLogGroups,
+  applyEnvVariables,
+  validateProps,
+  TagKeys,
+  DatadogProps,
+  DatadogStrictProps,
+  handleSettingPropDefaults,
+} from "./index";
+
+const versionJson = require("../version.json");
+
+export class Datadog extends cdk.Construct {
+  scope: cdk.Construct;
+  props: DatadogProps;
+  transport: Transport;
+  constructor(scope: cdk.Construct, id: string, props: DatadogProps) {
+    if (process.env.DD_CONSTRUCT_DEBUG_LOGS == "true") log.setLevel("debug");
```

Comment:
maybe we should lowercase before checking for equality?
we usually don't use single line in `if`, maybe it's better to use `{}` for readability here

Reply:
resolved using `{}` and lowercasing